### DISCUSSION TITLE
How to change background highlight ?

### DISCUSSION THREAD

**Discussion by radvil** [hedyhli/outline.nvim#61] [Help] [answered]
It seems like there is no default highlight for the background. If there is, what is the name of the highlight ?

**Comment by hedyhli** [comment, marked as answer]
There isn't a special highlight for the outline window. What you can do instead is set your own highlight group, say `OutlineWindow` with a special background color, then use this in your config:
```lua
outline_window = {
  -- ...
  winhl = "Normal:OutlineWindow",
},
```
